src/ducks/lagret-filter-ui-state.ts

```diff
@@ -0,0 +1,111 @@
+import {OrNothing} from "../utils/types/types";
+import {LagretFilter} from "./lagretFilter";
+
+// Actions
+export const VELG_MINE_FILTER = 'lagretfilter_velg/VELG_MINE_FILTER';
+export const MARKER_MINE_FILTER = 'lagretfilter_velg/MARKER_MINE_FILTER';
+export const AVMARKER_MINE_FILTER = 'lagretfilter_velg/AVMARKER_MINE_FILTER';
+export const AVMARKER_SISTE_VALGT_MINE_FILTER = 'lagretfilter_velg/AVMARKER_SISTE_VALGT_MINE_FILTER';
+
+export const MARKER_VEILEDER_GRUPPE = 'lagretfilter_velg/MARKER_VEILEDER_GRUPPE';
```

Comment:
Where are these: 
`VELG_VEILEDER_GRUPPE`
`APNE_VEILEDERGRUPPE_MODAL`
`LUKK_VEILEDERGRUPPE_MODAL`

Reply:
I can make the similar logic for veiledergruppe modal as for mine filter...regarding VELG_VEILEDER_GRUPPE this is done by https://github.com/navikt/veilarbportefoljeflatefs/blob/e7e5ca971f303ed3fbb74662b030c931cb71bae1/src/filtrering/filtrering-veileder-grupper/ny_veileder_gruppe_rad.tsx#L28

Reply:
Yes, I think we should be consistent so things are similar. Maybe add all of the same logic to `lagret-filter-ui-state.ts`

Reply:
Deal

Reply:
Only this left, and I'm happy with the refactoring :D

Reply:
Yep, me too...I am working on it :)

Reply:
It will be done today